starter/app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Location Reminders</string>
+    <string name="app_name">AAA Location Reminders</string>
```

Comment:
@graffiti75 Could explain why did you add AAA, please?

Reply:
New PR created: https://github.com/udacity/nd940-android-kotlin-c4-starter/pull/14